packages/snip20/src/handle.rs

```diff
@@ -0,0 +1,562 @@
+        callback_code_hash: String,
+        contract_addr: HumanAddr,
+        send_amount: Option<Uint128>,
+    ) -> StdResult<CosmosMsg> {
+        // can not have block size of 0
+        if block_size == 0 {
+            block_size = 1;
+        }
+        let mut msg = to_binary(self)?;
+        space_pad(&mut msg.0, block_size);
+        let mut send = Vec::new();
+        if let Some(amount) = send_amount {
+            send.push(Coin {
+                amount,
+                denom: String::from("uscrt"),
+            });
+        }
+        let execute = WasmMsg::Execute {
+            msg,
+            contract_addr,
+            callback_code_hash,
+            send,
+        };
+        Ok(execute.into())
+    }
+}
+
+/// Returns a StdResult<CosmosMsg> used to execute Redeem
+///
+/// # Arguments
+///
+/// * `amount` - Uint128 amount of token to redeem for SCRT
+/// * `denom` - Optional String to hold the denomination of tokens to redeem
+/// * `padding` - Optional String used as padding if you don't want to use block padding
+/// * `block_size` - pad the message to blocks of this size
+/// * `callback_code_hash` - String holding the code hash of the contract being called
+/// * `contract_addr` - address of the contract being called
+pub fn redeem_msg(
+    amount: Uint128,
+    denom: Option<String>,
+    padding: Option<String>,
+    block_size: usize,
+    callback_code_hash: String,
+    contract_addr: HumanAddr,
+) -> StdResult<CosmosMsg> {
+    HandleMsg::Redeem {
+        amount,
+        denom,
+        padding,
+    }
+    .to_cosmos_msg(block_size, callback_code_hash, contract_addr, None)
+}
+
+/// Returns a StdResult<CosmosMsg> used to execute Deposit
+///
+/// # Arguments
+///
+/// * `amount` - Uint128 amount of uSCRT to convert to the SNIP20 token
+/// * `padding` - Optional String used as padding if you don't want to use block padding
+/// * `block_size` - pad the message to blocks of this size
+/// * `callback_code_hash` - String holding the code hash of the contract being called
+/// * `contract_addr` - address of the contract being called
+pub fn deposit_msg(
+    amount: Uint128,
+    padding: Option<String>,
+    block_size: usize,
+    callback_code_hash: String,
+    contract_addr: HumanAddr,
+) -> StdResult<CosmosMsg> {
+    HandleMsg::Deposit { padding }.to_cosmos_msg(
+        block_size,
+        callback_code_hash,
+        contract_addr,
+        Some(amount),
+    )
+}
+
+/// Returns a StdResult<CosmosMsg> used to execute Transfer
+///
+/// # Arguments
+///
+/// * `recipient` - a reference to the address the tokens are to be sent to
+/// * `amount` - Uint128 amount of tokens to send
+/// * `padding` - Optional String used as padding if you don't want to use block padding
+/// * `block_size` - pad the message to blocks of this size
+/// * `callback_code_hash` - String holding the code hash of the contract being called
+/// * `contract_addr` - address of the contract being called
```

Comment:
Just want to point out you have exemplary good docstrings in this PR :sparkles: :100: :sparkles: